### PR TITLE
Add cert SAN for WC APIs

### PR DESCRIPTION
**Pull request by pipo02mix**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
 
+### Added
 
-## [Unreleased]
+- Add API server`certSANs` property to generate a certificate for the API FQDN.
```
Comment (tfussell) [inline CHANGELOG.md:12]:
```suggestion
- Add `certSANs` property to generate a certificate containing the Kubernetes API FQDN.
```

File: helm/cluster-openstack/templates/_kubeadm_control_plane_template.tpl [hunk 2]
```diff
@@ -12,6 +12,8 @@ spec:
       kubeadmConfigSpec:
         clusterConfiguration:
           apiServer:
+            certSANs:
+              - "api.{{ include "resource.default.name" $ }}.{{ .Values.baseDomain }}"
```
Comment (kopiczko) [inline helm/cluster-openstack/templates/_kubeadm_control_plane_template.tpl:16]: FMI: what's the difference between `{{ include "resource.default.name" $ }}` and `{{ include "resource.default.name" . }}` ?

**Review by kopiczko — APPROVED** [review]
LGTM with @tfussell is happy

File: helm/cluster-openstack/templates/_kubeadm_control_plane_template.tpl (hunk 2 above)
Comment (tfussell) [reply helm/cluster-openstack/templates/_kubeadm_control_plane_template.tpl:16]: > FMI: what's the difference between `{{ include "resource.default.name" $ }}` and `{{ include "resource.default.name" . }}` ?

`.` is the current scope (like if you use `{{ with ... }}`). `$` is the root scope which is the same everywhere.

**Pull request merged**